src/app/components/common/FundingSummary/FundingSummary.tsx

```diff
@@ -20,77 +18,73 @@ type Props = {
   liquidationCollateralCost?: number | Decimal,
 };
 
+const FundingSummeryColumns = {
+  FUNDING_SUMMARY: 'Funding Summary',
+  FEE: 'Fee (365 Days)',
+  VALIDATORS: 'Validators',
+  SUBTOTAL: 'Subtotal',
+};
+
 const FundingSummary = (props: Props) => {
-  const stores = useStores();
-  const classes = useStyles();
-  const ssvStore: SsvStore = stores.SSV;
-  const processStore: ProcessStore = stores.Process;
-  const operatorStore: OperatorStore = stores.Operator;
-  const process: RegisterValidator = processStore.process as RegisterValidator;
-  const daysPeriod = props.days ?? process.fundingPeriod;
-  const payments = [
-    { id: 1, name: 'Operator fee' },
-    { id: 2, name: 'Network fee' },
-    { id: 3, name: 'Liquidation collateral' },
-  ];
+    const stores = useStores();
+    const classes = useStyles();
+    const ssvStore: SsvStore = stores.SSV;
+    const processStore: ProcessStore = stores.Process;
+    const operatorStore: OperatorStore = stores.Operator;
+    const validatorStore: ValidatorStore = stores.Validator;
+    const process: RegisterValidator = processStore.process as RegisterValidator;
+    const daysPeriod = props.days ?? process.fundingPeriod;
+    const payments = [
+      { id: 1, name: 'Operator fee' },
+      { id: 2, name: 'Network fee' },
+      { id: 3, name: 'Liquidation collateral' },
+    ];
+
+    const networkCost = props.networkCost ?? propertyCostByPeriod(ssvStore.networkFee, daysPeriod);
+    const operatorsCost = props.operatorsCost ?? propertyCostByPeriod(operatorStore.getSelectedOperatorsFee, daysPeriod);
+    const liquidationCollateralCost = props.liquidationCollateralCost ?? new Decimal(operatorStore.getSelectedOperatorsFee).add(ssvStore.networkFee).mul(ssvStore.liquidationCollateralPeriod);
+
+    const paymentsValue = (paymentId: number | string): string => {
+      switch (Number(paymentId)) {
+        case 1:
```

Comment:
Use some enum and move it to ./models instead of using raw numbers

Reply:
will change in next PR